SDK/MobileCenter/Microsoft.Azure.Mobile.UWP/MobileCenterPart.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace Microsoft.Azure.Mobile
+﻿using Microsoft.Azure.Mobile.Utils;
+
+namespace Microsoft.Azure.Mobile
 {
     public partial class MobileCenter
     {
         private const string PlatformIdentifier = "uwp";
+
+        /// <summary>
+        /// Sets the device country to display in the portal
+        /// </summary>
+        /// <param name="country">The current country</param>
+        public static void SetCountry(string country)
```

Comment:
>SetCountry [](start = 27, length = 10)

I am not sure if rename it to 'deviceCountry' or 'carrierCountry' instead of 'country' would be more informative and consistent. Unless the contract requires 'country' as the property name, I feel more comfortable to use a longer name.